services/taxtobin/TaxTobinAspCore/TaxTobin/TaxTobin/Views/_ViewImports.cshtml

```diff
@@ -0,0 +1,2 @@
+﻿@using TaxTobin
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
```

Comment:
Ikke sikker på om filen er autogen eller ej. Hvis du har lavet den så tilføj manglende newline to sidst i filen. Så bliver git glad igen :)